frontend_tests/node_tests/compose.js

```diff
@@ -100,6 +100,15 @@ function test_ui(label, f) {
     run_test(label, f);
 }
 
+function mock_stream_header_colorblock() {
```

Comment:
This function appears in three files -- where's a good shared place I could put it?

Reply:
I think you can make a `compose_lib` file in `frontend_tests/node_tests/lib/` for this.

Reply:
Great, did that!